### PR TITLE
Remove icons in standalone search

### PR DESCRIPTION
**Pull request by andvirga**
- Removed `MoreHorizIcon` (...) in all scenarios
- Removed `LinkIcon` for Standalone search scenario.
- Fixed Flex reboot issue by chaging `<form>` wrapper to a `<div>`

### CODE REVIEW

**Review by nickhurlburt — CHANGES_REQUESTED** [review]
Mostly looking good, but some major issues.  Biggest thing is that whenever I do an in-contact search, then click into the cases search tab, Flex gets an error and forces a hard refresh.  Several errors and warnings get reported when this happens.  Please test out in-contact search to see this.

File: plugin-hrm-form/src/components/ContactDetails.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { format } from 'date-fns';
-import { ButtonBase, IconButton } from '@material-ui/core';
-import { MoreHoriz, Link as LinkIcon } from '@material-ui/icons';
+import { IconButton } from '@material-ui/core';
+import { Link as LinkIcon } from '@material-ui/icons';
 
 import { DetailsContainer, NameContainer, DetNameText, ContactDetailsIcon } from '../styles/search';
```
[the remark this answers is not in the document]
Comment (andvirga) [reply plugin-hrm-form/src/components/ContactDetails.tsx:7]: Good catch! Done!

File: plugin-hrm-form/src/components/search/ContactPreview/ConnectContact.tsx [hunk 2]
```diff
@@ -27,7 +28,7 @@ const ConnectContact: React.FC<Props> = ({ callType, onOpenConnectDialog }) => {
         <HiddenText>
           <Template code="ContactPreview-CopyButton" />
         </HiddenText>
-        <ConnectIcon />
+        {showConnectIcon && <ConnectIcon />}
```
[the remark this answers is not in the document]
Comment (andvirga) [reply plugin-hrm-form/src/components/search/ContactPreview/ConnectContact.tsx]: Solved!

**Comment by andvirga** [comment]
@nickhurlburt the flex reboot issue is solved now. (I've removed the `<form>` wrapper and replaced it with a `<div role='form'>` like @GPaoloni instructed.

**Review by nickhurlburt — APPROVED** [review]
Looks good to me!

**Review by murilovmachado — APPROVED** [review]
Looks good!

**Pull request merged**